syntax = "proto3";

package message;
service verify_service {
  rpc get_verify_code (get_verify_req) returns (get_verify_rsp) {}
}
message get_verify_req {
  string email = 1;
}
message get_verify_rsp {
  int32 error = 1;
  string email = 2;
  string code = 3;
}


message GetChatServerReq {
  int32 uid = 1;
}

message GetChatServerRsp {
  int32 error = 1;
  string host = 2;
  string port = 3;
  string token = 4;
}

message LoginReq{
	int32 uid = 1;
	string token= 2;
}

message LoginRsp {
	int32 error = 1;
	int32 uid = 2;
	string token = 3;
}

service StatusService {
	rpc GetChatServer (GetChatServerReq) returns (GetChatServerRsp) {}
	rpc Login(LoginReq) returns(LoginRsp);
}